File: src/main/java/net/hectus/neobb/turn/default_game/structure/glass_wall/TGreenGlassWall.java
package net.hectus.neobb.turn.default_game.structure.glass_wall;

import net.hectus.neobb.player.NeoPlayer;
import net.hectus.neobb.structure.PlacedStructure;
import net.hectus.neobb.turn.default_game.attributes.clazz.NatureClazz;

public class TGreenGlassWall extends GlassWallTurn implements NatureClazz {
    public TGreenGlassWall(NeoPlayer player) { super(player); }
    public TGreenGlassWall(PlacedStructure data, NeoPlayer player) { super(data, player); }

    @Override
    public String color() {
        return "green";
    }
}

File: src/main/kotlin/net/hectus/neobb/modes/turn/default_game/structure/glass_wall/TGreenGlassWall.kt
package net.hectus.neobb.modes.turn.default_game.structure.glass_wall

import com.marcpg.libpg.storing.Cord
import net.hectus.neobb.matrix.structure.PlacedStructure
import net.hectus.neobb.matrix.structure.StaticStructure
import net.hectus.neobb.matrix.structure.StaticStructures
import net.hectus.neobb.modes.turn.default_game.attribute.clazz.NatureClazz
import net.hectus.neobb.player.NeoPlayer

class TGreenGlassWall(data: PlacedStructure?, cord: Cord?, player: NeoPlayer?) : GlassWallTurn(data, cord, player), NatureClazz {
    override val staticStructure: StaticStructure = StaticStructures.Default.GlassWall.GREEN
}
